nbs/examples/Introduction.ipynb

```diff
@@ -0,0 +1,840 @@
+{
```

Comment:
<p>I think it would be a good complement to define and explain S_df and tags, and make the connection of the previous graphical examplanation to Y_dier_df. </p><p>Something like: The previously introduce hierarchical graph, where the top levels are the sum of the bottom levels will be called: <code>Y_hier_df</code> . For the reconciliation method we also need: <code>S_df</code> and <code>tags</code>. The S_df is... and the tags are....   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/hierarchicalforecast/pull/102/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>